src/main/java/com/devonfw/quarkus/productmanagement/logic/UcDelete.java

```diff
@@ -0,0 +1,20 @@
+package com.devonfw.quarkus.productmanagement.logic;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.transaction.Transactional;
+
+import com.devonfw.quarkus.productmanagement.domain.repo.ProductRepository;
+
+@Named
```

Comment:
Same for all other "singletons". Can you also test the app?
Or did you test it and it is working since you found some magic trick to make quarkus behave like spring?

Reply:
it was tested and worked well. Anyway changing to @ApplicationScoped